src/content/overview/architecture/_index.md

```diff
@@ -11,7 +11,57 @@ user_questions:
   - What is the architecture of the Giant Swarm cloud-native developer platform?
 owner:
   - https://github.com/orgs/giantswarm/teams/team-planeteers
-last_review_date: 2024-04-26
+last_review_date: 2024-11-04
 ---
 
-## Platform API {#platform-api}
+Giant Swarm's cloud-native platform is a collection of open-source components that work together to provide a seamless experience for managing the lifecycle of containerized applications. The platform is designed to be cloud-agnostic, allowing you to deploy your applications on any of the supported cloud providers. Besides, the platform offers a rich set of APIs that our customers can use to offer a set of self-service capabilities to their developers.
+
+## Platform architecture
+
+![Platform architecture](./platform-architecture.png)
+
+The platform architecture consists of the following layers:
+
+- Infrastructure layer: responsible for managing the underlying infrastructure, including the cloud provider resources or on-premises infrastructure. To abstract the underlying infrastructure, the platform uses the `Kubernetes`.
+
+- Capabilities layer: provides a set of capabilities to manage the lifecycle of applications, including deploying, scaling, and monitoring applications.
+
+- Interface layer: offers a set of templates, APIs, and tools that developers can use to interact with the platform.
+
+Along these layers, Giant Swarm provides a real support channel, incident management, and good documentation to help you get the most out of the platform. Additionally, you have a dedicated account engineer to help creating a common roadmap and to provide guidance on how to use the platform.
+
+## Platform API
+
+The Giant Swarm platform is build on top of `Kubernetes` thanks to [`Cluster API`](https://cluster-api.sigs.k8s.io/), a open source  project that standardize the cluster lifecycle management across different cloud providers or on-premises infrastructure.
+
+In `Cluster API`, there is a distinction between the management cluster and the workload clusters. The management cluster is responsible for managing the lifecycle of the workload clusters. The workload clusters are the clusters where you deploy your applications.
```

Comment:
I'm not sure if i'd go into MC/WC topology here somehow, we don't mention it in the diagram

Reply:
Hmm, I wanted to explain what the platform really is and the platform API. It is hard without introducing the concept of MC/WC. At the same time, all installation flavours have an MC/WC concept, though they could vary in form. I think we could go deeper in child documents explaining the different setups but introduce here the concept IMO is valid :D

Reply:
Maybe don't go into details that much, but still keep the sentence:

```suggestion
The Platform API is the regular Kubernetes API of your central _management cluster_ and is your interface for deploying workload clusters and applications, or to reach other capabilities such as the observability and security dashboards.
```